AsyncDisplayKit/Details/ASRangeController.h

```diff
@@ -77,6 +82,15 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+@protocol ASRangeControllerUpdateRangeProtocol <NSObject>
```

Comment:
This is an interesting idea, but I am a bit concerned about introducing it now, At least this public API. Could you implement this as private API for now, and still have the functionality integration with ASViewController work?  You could put it in a +Beta.h header somewhere.

The reason is that I am planning to add a similar, but larger and differently main protocol to include things like interface state and potentially a few other capabilities that need to be shared between nodes and controllers - or between collection node / collection view and the table versions.  Ultimately I think you have the right intuition here, I am simply interested in ensuring that the API naming remains consistent and stable as we add this functionality in the future